### PR TITLE
Updated UG and help message

### LINKED ISSUE #62: Update help command details
Currently, the help command prints
> Refer to the user guide: https://se-education.org/addressbook-level3/UserGuide.html

Needs to be re-set to the correct URL, and also more changes might be required to the help command.

### PR DESCRIPTION
**Pull request by takufunkai**
Fixes #62 and part of #82

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -2,9 +2,15 @@
...
 
-d'Intérieur is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still
-having the benefits of a Graphical User Interface (GUI). If you can type fast, d'Intérieur can get your contact
-management tasks done faster than traditional GUI apps.
+d'Intérieur is a **desktop app for interior designers to manage their contacts and projects, optimized for use via a 
+Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). 
+If you can type fast, d'Intérieur can get your contact management tasks done faster than traditional GUI apps.
+
+If you utilize other online applications for interior designing and lack a centralized, robust, and efficient customer
+management tool, this application will be able to meet your needs.
```
Comment (glennljw) [inline docs/UserGuide.md:10]: Great job on capturing the limitations of online applications which can be addressed with d'Intérieur!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -177,7 +198,7 @@ Examples:
 
 ### Locating contacts by tag : `findtag`
 
-Find contact with the given tag.
+Finds contact with the given tag.
```
Comment (glennljw) [inline docs/UserGuide.md:201]: Nice! Thanks for spotting this mistake!

File: docs/UserGuide.md [hunk 3]
```diff
@@ -140,7 +146,22 @@ Examples:
...
+![unfavourited](images/after_unfavourite_command.png)
+
+
+### Listing Favourites : `favourites`
+
+Lists all your favourite contacts to the list of displayed contacts.
+
+Format:
+
+- `favourites`
```
Comment (glennljw) [inline docs/UserGuide.md:162]: Same as the above mentioned

File: docs/UserGuide.md [hunk 4]
```diff
@@ -125,12 +131,12 @@ Examples:
 
 ### Adding favourites : `fav`
 
-Adds the contact to your list of favourite contacts and view them.
+Toggles the favourite status of your contacts. 
+Favourited contacts **can be un-favourited** by running the same command on the contact again.
 
 Format:
 
 - `fav INDEX`
```
Comment (glennljw) [inline docs/UserGuide.md:139]: Might want to keep this to 1 line, e.g. `Format: fav INDEX` to be consistent with the other features. Just a suggestion!

File: docs/UserGuide.md [hunk 5]
```diff
@@ -140,7 +146,22 @@ Examples:
 
 - `fav 1` — Adds contact at index 1 to your list of favourites
 
-### Adding high importance flag: `impt`
+![favourite](images/favourite_command.png)
+
+- `fav 1` - Run the command for the same person, and the favourite status will be toggled off.
```
Comment (glennljw) [inline docs/UserGuide.md:151]: Should we be using `contacts` instead of `person` over here?

**Review by glennljw — CHANGES_REQUESTED** [review]
LGTM! Just a few nits to fix! Overall great job on keeping the consistency the same and spotting mistakes!

File: docs/UserGuide.md (hunk 5 above)
Comment (takufunkai) [reply docs/UserGuide.md:151]: Thanks!

**Review by glennljw — APPROVED** [review]

**Pull request merged**